app/routes/screens/clubFeed/helper.js

```diff
@@ -0,0 +1,27 @@
+export const getIconName = (value = "") => {
```

Comment:
try to use a mapping object instead of plain switch case 

return mappingObject[value]?mappingObject[value]:"error";